iati/core/resources.py

```diff
@@ -191,6 +201,42 @@ def get_codelist_path(codelist_name, version=None):
     return get_path_for_version(os.path.join(PATH_CODELISTS, '{0}'.format(codelist_name) + FILE_CODELIST_EXTENSION), version)
 
 
+def get_codelist_mapping_path(version=None):
+    """Determine the path of the Codelist mapping file.
+
+    version (str): The version of the Standard to return the data files for. Defaults to None. This means that the path is returned for a filename at the latest version of the Standard.
+
+    Returns:
+        str: The path to a file containing the mapping file.
+
+    Todo:
+        Test this.
```

Comment:
Can be tested with the `*standard_version_optional` fixture: https://github.com/IATI/iati.core/blob/dev/iati/core/tests/utilities.py#L131-L144

Reply:
These various things in `resources.py` are present in the `dev` branch - suggest that they should be fixed in a branch forked from there.

Reply:
Sounds sensible.